language/move-vm/runtime/src/runtime.rs

```diff
@@ -172,7 +172,12 @@ impl VMRuntime {
         // all the checks, then the whole bundle can be published/upgraded together. Otherwise,
         // none of the module can be published/updated.
         for (module, blob) in compiled_modules.into_iter().zip(modules.into_iter()) {
-            data_store.publish_module(&module.self_id(), blob)?;
+            let runtime_id = module.self_id();
```

Comment:
out of curiosity and meant to ask before too, why `runtime_id`? I would have used `module_id` as that seems a pretty "stable" concept, I am actually wondering if I am missing something now....
because I am thinking I am not sure anymore what we wrote in the binary for the module id...
commit happily but let's make sure to talk tomorrow about this